### ISSUE TITLE
Error on relationship or select columns with custom attribute as value

### ISSUE THREAD

**Issue by bastos71** [Laravel-Backpack/CRUD#2813]
# Bug report

### What I did

Try to use a combined attribute as `select` or `relationship`  column value

Example : 
```php
// MyCrudController.php
$this->crud->addColumn([
    'name' => 'customer',
    'type' => 'relationship',
    'label' => 'Customer Full Name',
    'attribute' => 'full_name',
]);

// Customer.php
public function getFullNameAttribute(): string
{
    return sprintf('%s %s', $this->first_name, $$this->last_name);
}
```

### What I expected to happen

Show me the value :)

### What happened

Throwed an error :(

### What I've already tried to fix it

Add my custom combined attribute in the `$append` entitie's property.
```php
// Customer.php
protected $appends = [
    'full_name',
];
```

### Backpack, Laravel, PHP, DB version

When I run ```php artisan backpack:version``` the output is:

```
### PHP VERSION:
PHP 7.3.13 (cli) (built: Dec 18 2019 16:08:06) ( NTS )
Copyright (c) 1997-2018 The PHP Group
Zend Engine v3.3.13, Copyright (c) 1998-2018 Zend Technologies
    with Zend OPcache v7.3.13, Copyright (c) 1999-2018, by Zend Technologies

### LARAVEL VERSION:
v6.18.13@39519d03237a81380e2126d667e898e2922db444

### BACKPACK VERSION:
4.1.2@7084182b38bff6b19624788ff143bb249ecdc758
```

**Comment by pxpm** [comment]
Hello @bastos71 

Sorry for introducing this change without documenting it. Well, i missed it, that's why it's not documented :disappointed: 

It's related to #2810 

I'v fixed it with #2815 

So I am going to close this.

Thanks for report :)

**Comment by bastos71** [comment]
Thx for your answer ! 
My bad, I did not see the previous issue
